src/PhoneNumber/PhoneNumber.php

```diff
@@ -0,0 +1,69 @@
+<?php
+declare(strict_types=1);
+
+namespace DR\Internationalization\PhoneNumber;
+
+use libphonenumber\PhoneNumber as LibPhoneNumber;
+
+class PhoneNumber
+{
+    /**
+     * @param string $countryCode 2 Character ISO country code (NL, BE, FR, ES)
+     *
+     * @internal Only receive this object from the PhoneNumberParseService::parse function.
+     */
+    public function __construct(
+        private string $internationalDialCode,
+        private string $countryDialCode,
+        private string $nationalNumber,
+        private string $rawInput,
+        private string $countryCode,
+        private PhoneNumberTypeEnum $numberType,
+        private LibPhoneNumber $phoneNumber,
+        private ?string $extension = null
+    ) {
+    }
+
+    public function getInternationalDialCode(): string
```

Comment:
Could you add some docblocks with a description to these methods. What do these values mean? Maybe with an example?